src/librustc_mir/borrow_check/mod.rs

```diff
@@ -1260,6 +1260,31 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
                 );
             }
 
+
+            Rvalue::AddressOf(mutability, ref place) => {
+                let access_kind = match mutability {
```

Comment:
This is repeated in `invalidation.rs` -- should be de-duped with a method.

Reply:
The whole of `invalidation.rs` should be deduplicated. I don't really see the point of doing it for the one thing that happens to be in this PR.

Reply:
Ah; I had no idea :)